src/core-files.json

```diff
@@ -495,8 +493,6 @@
     "private_headers": {
         "csscolorparser/csscolorparser.hpp": "src/csscolorparser/csscolorparser.hpp",
         "mbgl/algorithm/covered_by_children.hpp": "src/mbgl/algorithm/covered_by_children.hpp",
```

Comment:
Could also mbgl/algorithm/covered_by_children.hpp and test/algorithm/covered_by_children.test.cpp be removed?

Reply:
Yup, removed!